.travis.yml

```diff
@@ -12,4 +12,6 @@ addons:
 install:
   - pip install pytest-travis-fold
   - make test-setup doc-setup
```

Comment:
Do you want to save time and skip doc-setup on Python 2.7 here?

Reply:
good idea, done